java/gazelle/private/testutil/reset_zerolog_timestamps/dummy_language.go

```diff
@@ -0,0 +1,13 @@
+package reset_zerolog_timestamps
```

Comment:
Nit: Could you rename this file to "placeholder_language" or "noop_language" (for inclusive language)